code/game/jobs/access.dm

```diff
@@ -258,24 +258,26 @@ proc/get_all_job_icons() return joblist + list("Prisoner")//For all existing HUD
 		if("E4") . = size ? "CPL " : "Corporal"
 		if("E5") . = size ? "SGT " : "Sergeant"
 		if("E6") . = size ? "SSGT " : "Staff Sergeant"
-		if("E6E") . = size ? "TSGT " : "Technical Sergeant"
+		if("E6T") . = size ? "TSGT " : "Technical Sergeant"
 		if("E7") . = size ? "SFC " : "Sergeant First Class"
+		if("E7E") . = size ? "GSGT " : "Gunnery Sergeant"
 		if("E8") . = size ? "MSGT " : "Master Sergeant"
 		if("E8E") . = size ? "FSGT " : "First Sergeant"
 		if("E9") . = size ? "SGM " : "Sergeant Major"
 		if("E9E") . = size ? "CSGM " : "Command Sergeant Major"
 		if("O1") . = size ? "ENS " : "Ensign"
+		if("O1E") . = size ? "2LT " : "Second Lieutenant"
```

Comment:
There's First Lieutenant (1LT) and Lieutenant Junior Grade (LTJG)

Reply:
Is there?
I can't think of a single military that uses LTJG->1LT.
It usually goes one of the following:
- 2LT->LT
- 2LT ->1LT
- SLT (Sub-Lieutenant) -> LT
- LT(JG) -> LT(SG)

Reply:
The US Navy uses LTJG for this rank, or Sub-Lieutenant. Anything works here, really.